Competition2017/src/competition/subsystems/feeder/SideFeederSubsystem.java

```diff
@@ -0,0 +1,170 @@
+package competition.subsystems.feeder;
+
+import org.apache.log4j.Logger;
+import com.ctre.CANTalon.FeedbackDevice;
+import com.ctre.CANTalon.TalonControlMode;
+import edu.wpi.first.wpilibj.Timer;
+import xbot.common.command.BaseSubsystem;
+import xbot.common.controls.actuators.XCANTalon;
+import xbot.common.injection.wpi_factories.WPIFactory;
+import xbot.common.properties.BooleanProperty;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+
+public class SideFeederSubsystem extends BaseSubsystem {
+
+    private static Logger log = Logger.getLogger(FeederSubsystem.class);
+
+    private final XCANTalon masterMotor;
+
+    public enum FeederSide {
+
+        Left, 
+
+        Right
+    }
+
+    private final FeederSide side;
+    
+    private final DoubleProperty feederTestingPowerStep;
+    private final DoubleProperty feederCurrentSpeed;
+    private final DoubleProperty feederTargetSpeed;
+    private final DoubleProperty feederOutputPower;
+    private final DoubleProperty feederTalonError;
+    
+    private final BooleanProperty atSpeedProp;
+    private final BooleanProperty enableFeederLogging;
+
+    private final DoubleProperty p;
+    private final DoubleProperty i;
+    private final DoubleProperty d;
+    private final DoubleProperty f;
+
+    private final DoubleProperty feederSpeedThresh;
+
+    public SideFeederSubsystem(int motor, FeederSide side, WPIFactory factory, XPropertyManager propManager){
+
+        log.info("Creating " + side + " Feeder");
+
+        this.side = side;
+
+        p = propManager.createPersistentProperty(side + "Feeder wheel P", 0);
+        i = propManager.createPersistentProperty(side + "Feeder wheel I", 0);
+        d = propManager.createPersistentProperty(side + "Feeder wheel D", 0);
+        f = propManager.createPersistentProperty(side + "Feeder wheel F", 0);
+
+        feederTestingPowerStep = propManager.createPersistentProperty(side + "Feeder power step", 0.025);
+        feederSpeedThresh = propManager.createPersistentProperty(side + "Feeder nominal speed thresh (TPC)", motor);
+        feederCurrentSpeed = propManager.createEphemeralProperty(side + "Feeder current speed (TPC)", 0);
+        feederTargetSpeed = propManager.createEphemeralProperty(side + "Feeder goal speed (TPC)", 0);
+        feederOutputPower = propManager.createEphemeralProperty(side + "Feeder voltage", 0);
+
+        atSpeedProp = propManager.createEphemeralProperty("Is" + side + "Feeder at speed?", false);
+        feederTalonError = propManager.createEphemeralProperty(side + "Feeder speed error", 0);
+        enableFeederLogging = propManager.createEphemeralProperty("Is " + side + "Feeder logging info?", false);
+
+        this.masterMotor = factory.getCANTalonSpeedController(motor);
+
+        setUpMotorTeam(masterMotor);
```

Comment:
same thing here, this was taken from the `DriveSubsystem` where it sets up a 'pair' of motors but here there's just 1 so we can just put the setup of the motor inside the constructor or otherwise rename the method something like `setUpMotor`